### DISCUSSION TITLE
收集了某领域内的语料库，利用ChatGLM3进行增量预训练时，数据格式应该是怎么样的？这种无监督的数据格式有什么特殊要求吗？

### DISCUSSION THREAD

**Discussion by Ye-XX** [zai-org/ChatGLM3#982] [Q&A]
目前微调时候的数据格式是
{"conversations": [{"role": "user", "content": "类型#裤*版型#宽松*风格#性感*图案#线条*裤型#阔腿裤"}, {"role": "assistant", "content": "宽松的阔腿裤这两年真的吸粉不少，明星时尚达人的心头爱。毕竟好穿时尚，谁都能穿出腿长2米的效果宽松的裤腿，当然是遮肉小能手啊。上身随性自然不拘束，面料亲肤舒适贴身体验感棒棒哒。系带部分增加设计看点，还让单品的设计感更强。腿部线条若隐若现的，性感撩人。颜色敲温柔的，与裤子本身所呈现的风格有点反差萌。"}]}

那如果像将chatglm3-6b作为基座进行增量预训练的话，数据格式有什么要求？各位大佬们是怎么处理？

**Comment by Ye-XX** [comment]
诚邀大家共同讨论。做垂直预训练的数据格式与微调的数据格式会有那些不同？

**Reply by zzb1185** [reply]
请问问题解决了吗

**Reply by Ye-XX** [reply]
可以参考LLaMA-Factory框架进行

**Comment by zzb1185** [comment]
请问解决了吗？我也有这个问题呢

**Comment by zhaohai7206** [comment]
同求数据格式，有人做过这样训练的吗？
